### PR TITLE
Replaces nearly every mention of the SCCV Horizon being a Station with Ship/Vessel

### PR DESCRIPTION
**Pull request by Chada-1**
AFAIK the SCCV Horizon is a ship or vessel, it wouldn't be considered a Station, and as it stands lots of people are having problems moving away from the old terminology due to how everything on the map is named, so this is an attempt to rename everything to suit the new map faaar more, it hopefully doesn't break anything, but I need help making sure it doesn't.

### CODE REVIEW

File: code/game/machinery/computer/camera.dm [hunk 1]
```diff
@@ -257,7 +257,7 @@
 
 /obj/machinery/computer/security/wooden_tv
 	name = "security camera monitor"
-	desc = "An old TV hooked into the stations camera network."
+	desc = "An old TV hooked into the ships camera network."
```
Comment (Greenjoe12345) [inline code/game/machinery/computer/camera.dm:260]:
```suggestion
	desc = "An old TV hooked into the ship's camera network."
```

File: code/game/machinery/magnet.dm [hunk 2]
```diff
@@ -9,7 +9,7 @@
 	icon = 'icons/obj/objects.dmi'
 	icon_state = "floor_magnet-f"
 	name = "Electromagnetic Generator"
-	desc = "A device that uses station power to create points of magnetic energy."
+	desc = "A device that uses a vessels power to create points of magnetic energy."
```
Comment (Greenjoe12345) [inline code/game/machinery/magnet.dm:12]:
```suggestion
	desc = "A device that uses a vessel's power to create points of magnetic energy."
```

**Comment by alsoandanswer** [comment]
This should probably use the same system to determine station name and whatnot rather than be hardcoded.

File: code/game/machinery/newscaster.dm [hunk 3]
```diff
@@ -387,7 +387,7 @@ var/list/obj/machinery/newscaster/allCasters = list() //Global list that will co
 				dat+="<B>Wanted Issue successfully deleted from Circulation</B><BR>"
 				dat+="<BR><A href='?src=\ref[src];setScreen=[0]'>Return</A><BR>"
 			if(18)
-				dat+="<B><span class='boldannounce'>-- STATIONWIDE WANTED ISSUE --</B></span><BR><FONT SIZE=2>\[Submitted by: <span class='good'>[SSnews.wanted_issue.backup_author]</span>\]</font><HR>"
+				dat+="<B><span class='boldannounce'>-- VESSELWIDE WANTED ISSUE --</B></span><BR><FONT SIZE=2>\[Submitted by: <span class='good'>[SSnews.wanted_issue.backup_author]</span>\]</font><HR>"
```
Comment (Greenjoe12345) [inline code/game/machinery/newscaster.dm:390]: I feel like Shipwide sounds better then Vesselwide

**Comment by SleepyGemmy** [comment]
@Greenjoe12345 please stop posting one review per comment.

**Comment by Chada-1** [comment]
> This should probably use the same system to determine station name and whatnot rather than be hardcoded.

It does where it can, but it's just that using the 'Station name' variable only returns the full name of the map, not whether it's a ship or station, so it's kinda complicated and would require refactoring to do so, I'm down for it, but would need help.

File: code/game/machinery/newscaster.dm (hunk 3 above)
Comment (Chada-1) [reply code/game/machinery/newscaster.dm:390]: This is good, I can make sure it's shipwide across the files

File: code/game/objects/items/weapons/material/twohanded.dm [hunk 4]
```diff
@@ -335,7 +335,7 @@
 // Chainsaws!
 /obj/item/material/twohanded/chainsaw
 	name = "chainsaw"
-	desc = "A robust tree-cutting chainsaw intended to cut down various types of invasive spaceplants that grow on the station."
+	desc = "A robust tree-cutting chainsaw intended to cut down various types of invasive spaceplants that grow on various space stations."
```
Comment (Greenjoe12345) [inline code/game/objects/items/weapons/material/twohanded.dm:338]:
```suggestion
	desc = "A robust tree-cutting chainsaw intended to cut down various types of invasive space plants."
```
could likely just cut it down to this

**Comment by Chada-1** [comment]
> > This should probably use the same system to determine station name and whatnot rather than be hardcoded.
> 
> It does where it can, but it's just that using the 'Station name' variable only returns the full name of the map, not whether it's a ship or station, so it's kinda complicated and would require refactoring to do so, I'm down for it, but would need help.

Apparently there's a system for deciding if it's a ship or a station, so I'm going to go through all of the current hardcoded stuff and replace it with that system, then I'll set it up for review again

File: code/modules/mob/living/carbon/brain/posibrain.dm [hunk 5]
```diff
@@ -53,7 +53,7 @@
 
 	to_chat(brainmob, "<b>You are a positronic brain, brought into existence on [station_name()].</b>")
 	to_chat(brainmob, "<b>As a synthetic intelligence, you answer to all crewmembers, as well as the AI.</b>")
-	to_chat(brainmob, "<b>Remember, the purpose of your existence is to serve the crew and the station. Above all else, do no harm.</b>")
+	to_chat(brainmob, "<b>Remember, the purpose of your existence is to serve the crew and the vessel you were operated on. Above all else, do no harm.</b>")
```
Comment (Greenjoe12345) [inline code/modules/mob/living/carbon/brain/posibrain.dm:56]:
```suggestion
	to_chat(brainmob, "<b>Remember, the purpose of your existence is to serve the crew and the vessel you were activated on. Above all else, do no harm.</b>")
```

**Pull request closed without merging**